### ISSUE TITLE
Revamp of design for get.opensuse.org for the next-gen

### ISSUE THREAD

**Issue by lkocman** [openSUSE/get-o-o#124]
We're expecting a period of time when we'll be paralelly offering next-gen projects aside from current offerings.
If we add one or two more products, users will have quite hard time to figure out what do they want to download.

I think we want to provide an easy guidance to choose distribution that matches users intention.

![current](https://user-images.githubusercontent.com/510119/191835410-dc2e4b3f-90f3-4645-a021-f830678525e8.png)

@hellcp suggested that we could going the direction of https://contribute.opensuse.org/en#infrastructure
which is interactive, but would add quite some extra clicks to get to the download page.

I also like this approach (doesn't have to be honeycombs)
![distributions](https://user-images.githubusercontent.com/510119/191835501-a916cb14-eb68-4377-b614-4d09547503fc.svg)

**Comment by hellcp** [comment]
The initial intention of the get-o-o page was to be an interactive introduction, but it got intentionally trimmed down into less levels to make it easier on the users. I think still going with the desktop/server split makes sense, but we may need to be a bit more concise and try to divide next gen and current gen to point people who are the intended audience into the correct direction.

I don't see how even less information in case of honeycombs makes things any easier tbh. Also those logos look really wrong, why is leap micro and alp (wrong logo to begin with) so thick. While you are using new logos, may as well go for the new Leap and Tumbleweed logos too ;)

**Comment by lkocman** [comment]
I think desktop / generic server / Host OS (or another hype such as Os-as-a-Service or / ZeroTouch) and then just choose the pace (rolling vs regular release). That gives you three options and then you choose how fresh you want to be.

**Comment by hellcp** [comment]
Suggestions for more specialized pages are in #3, feel free to suggest a split there

**Comment by lkocman** [comment]
Good point @hellcp. Also people are raising the overlap in between landing page and get-o-o.

**Comment by Cameron-Cn** [comment]
I'm interested in helping with the design for get-o-o along with @sheimbach. 

Back at the openSUSE conference (this June in Nuremberg) we did some sketch drawings for potential updates to the look and logic of the main opensuse(.)org site.

The get-o-o could be edited along similar lines. I'll work on getting those doodle sketches (in a notebook, not a napkin thankfully) digitized into something a bit nicer and proposed here.

- Coming from the openSUSE weekly meeting (20220922)

**Comment by hellcp** [comment]
I would be interested in hearing more about www revamp in the issues of [openSUSE/www-o-o](https://github.com/openSUSE/www-o-o), since I have done some work on it already, and it would probably be better to try to sync up instead of having multiple different plans ;)

**Comment by sharabhh** [comment]
Hello, are any changes finalised that need to be made?
